_studio/shared/src/libmfx_core.cpp

```diff
@@ -1680,6 +1680,9 @@ bool CommonCORE::CheckOpaqueRequest(mfxFrameAllocRequest *request,
                                     mfxU32 NumOpaqueSurface,
                                     bool ExtendedSearch)
 {
+    if (!pOpaqueSurface || !request)
+        return false;
+
     if (pOpaqueSurface &&
```

Comment:
1683 will make sure pOpaqueSurface is not null, so pOpaqueSurface always true here. Better remove it from this if